src/WorkItemMigrator/JiraExport/JiraItem.cs

```diff
@@ -47,13 +47,9 @@ private static List<JiraRevision> BuildRevisions(JiraItem jiraItem, IJiraProvide
             if (createdOn == DateTime.MinValue)
                 Logger.Log(LogLevel.Debug, "created key was not found, using DateTime default value");
 
-
-            var changelog = jiraProvider.DownloadChangelog(issueKey).ToList();
```

Comment:
What happens for users using Jira server version 7?

Reply:
All versions treated equal. Jira Server 7 implements REST API v2 and should be work the same as Server 8. But I'm trusting the documentation here, I have not live tested it. See https://docs.atlassian.com/software/jira/docs/api/REST/7.0.0/#api/2/issue-getIssue and https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-issueidorkey-get